Add changelog

notthatbreezy opened the pull request:
Adds changelog

I messed up with pushing to develop earlier, so if you have comments about the current setup I can deal with those now too.

This image is basically designed to install dependencies for a django project that needs gunicorn and postgres/postgis support

<!-- Reviewable:start -->

[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/azavea/docker-django/1)

<!-- Reviewable:end -->

Commit:
Add changelog

hectcastro commented:
Grouping comments on `Dockerfile`:
- I think `python:2.7-slim` may be just as good a base for this container as `debian:jessie`
- I think that the `apt-get install` step should include the `--no-install-recommends` flag
- We may need to add `git` here at some point for `pip` dependencies that require it
- Should the dependencies go in a `requirements.txt` file?
- Some of the base Python images from Docker add the `--no-cache-dir` flag to `pip install`
- Should we add an `ENTRYPOINT` to the `gunicorn` binary?

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, all commit checks successful.

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/azavea/docker-django/1)

<!-- Sent from Reviewable.io -->

hectcastro commented:
Reviewed 1 of 1 files at r1.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/azavea/docker-django/1)

<!-- Sent from Reviewable.io -->

notthatbreezy commented:
All good comments, I'll try all of them out now.

Commit:
Changes based on feedback
 - install git
 - use requirements.txt
 - use --no-cache-dir option
 - use --no-install-recommends option
